gtfs-netex-test/gtfs_show_map.py

```diff
@@ -18,7 +18,7 @@ def generate_random_dark_color():
 
 
 def main(gtfs_zip_file, map_file, limitation):
-    global processing_data
+    #global processing_data
```

Comment:
Think this should be removed, if not required.

Reply:
ook